seqr/views/apis/summary_data_api.py

```diff
@@ -416,3 +436,119 @@ def _get_airtable_collaborator_names(user, collaborator_ids):
+
+    # Get a map of the family_ids associated with each individual in the report.
+    # A given individual may be in multiple projects.
+    individual_ids = [individual['metadata']['ext_id'] for individual in results.values()]
+    family_id_map = defaultdict(list)
+    all_family_ids = set()
+    for individual_id, family_id in Individual.objects.filter(
+        family__project__in=projects, individual_id__in=individual_ids,
+    ).values_list('individual_id', 'family_id'):
+        family_id_map[individual_id].append(family_id)
+        all_family_ids.add(family_id)
+
+    missing_individuals = set(individual_ids) - set(family_id_map.keys())
+    if missing_individuals:
+        raise ErrorsWarningsException([f'Unable to find the following individuals: {", ".join(sorted(missing_individuals))}'])
+
+    # Make a dict of (family_id, variant_id) -> AIP prediction for variant.
+    # Adds the variant multiple times if the family is in multiple projects.
+    all_variant_ids = set()
+    family_variant_data = {}
+    for sg_id, individual_aip_results in results.items():
+        individual_id = results[sg_id]['metadata']['ext_id']
+        family_ids = family_id_map[individual_id]
+        for family_id in family_ids:
+            for variant_result in individual_aip_results['variants']:
+                variant_id = variant_result['var_data']['info']['seqr_link']
+                family_variant_data[(family_id, variant_id)] = variant_result
+                all_variant_ids.add(variant_id)
+
+    # Get a map of the saved variants that are already in the database.
+    saved_variant_map = {
+        (v.family_id, v.variant_id): v
+        for v in SavedVariant.objects.filter(family_id__in=all_family_ids, variant_id__in=all_variant_ids)
+    }
+
+    # If the variant has not been "saved" before, find it in the search backend and save it.
+    new_variants = set(family_variant_data.keys()) - set(saved_variant_map.keys())
+    if new_variants:
+        new_variants_from_search = _search_new_saved_variants(new_variants, user, warnings)
+        saved_variant_map.update(new_variants_from_search)
+
+    # Add the aip_permissive tag to all variants
+    aip_tag_type = VariantTagType.objects.get(name='AIP-permissive', project=None)
+    num_new, num_updated = _cpg_add_aip_tags_to_saved_variants(aip_tag_type, saved_variant_map, family_variant_data, category_map, user, restrictive=False)
+
+    # Add the aip_restrictive tag to all variants
+    aip_restrictive_tag_type = VariantTagType.objects.get(name='AIP-restrictive', project=None)
+    num_new_restrictive, num_updated_restrictive = _cpg_add_aip_tags_to_saved_variants(aip_restrictive_tag_type, saved_variant_map, family_variant_data, category_map, user, restrictive=True)
+
+    summary_message = f'Loaded {num_new} new ({num_new_restrictive} restrictive) and {num_updated} updated ({num_updated_restrictive} restrictive) AIP tags for {len(family_id_map)} families'
+    safe_post_to_slack(
```

Comment:
Do we have this configured?

Reply:
Not to my knowledge. I just cloned the logic from the original function to maintain parity in case we ever did configure it.